Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```
@@ -43,6 +43,7 @@
     ClothingHeadHatHairflower: 1
     ClothingHeadHatGladiator: 1
     ClothingUniformJumpsuitGladiator: 1
+    ClothingUniformCostumeVelvetSkirt: 1
```

```suggestion
    ClothingUniformCostumeVelvetSkirt: 1 # DeltaV - Add madotsuki outfit
```
Remember, if the file you're modifying is an upstream file, we need there to be a comment explaining that it's a downstream change, and why it's there. This makes it so that when there's a change to this file from upstream, we're able to see how to best merge the change.